Github/legos/github_get_branch/github_get_branch.py

```diff
@@ -0,0 +1,74 @@
+##
+##  Copyright (c) 2021 unSkript, Inc
+##  All rights reserved.
+##
+from github import GithubException
+from typing import Optional, List
+from pydantic import BaseModel, Field
+import pprint
+
+
+class InputSchema(BaseModel):
+    branch_name: str = Field(
+        description='Branch name. Eg:"dummy-branch-name"', 
+        title='Branch Name'
+    )
+    owner: str = Field(
+        description='Username of the GitHub user. Eg: "johnwick"',
+        title='Owner'
+    )
+    repository: str = Field(
```

Comment:
Should this just be the repo?  Since the owner would have "unSkript"... is it redundant to ask for 
Owner: "unSkript"
Repo: "unskript/Awesome-CloudOps-Automations"